scripts/sync-react.js

```diff
@@ -292,24 +302,20 @@ Or, run this command with no arguments to use the most recently published versio
   )
 
   await sync({
-    newDateString,
-    newSha,
-    newVersionStr,
+    newVersionStr: `0.0.0-experimental-${newSha}-${newDateString}`,
     noInstall: !install,
     channel: 'experimental',
   })
   if (commit) {
     await commitEverything('Update `react@experimental`')
   }
   await sync({
-    newDateString,
-    newSha,
     newVersionStr,
     noInstall: !install,
-    channel: 'rc',
+    channel: '<framework-stable>',
```

Comment:
There isn't a meaningful channel here. It's whatever you passed in `--version` which is what we consider "framework stable".